build/common.props

```diff
@@ -45,6 +45,7 @@
   
   <ItemGroup Condition="'$(TargetFramework)' != 'net8.0'">
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="$(MicrosoftSourceLinkGitHubVersion)" PrivateAssets="All"/>
+    <PackageReference Include="Newtonsoft.Json" Version="$(NewtonsoftVersion)" />
```

Comment:
Does this code build if you are targeting `net8.0`? How does this PackageReference come in that case?

Reply:
I will have to fix that.